packages/cfpb-layout/src/molecules/card.less

```diff
@@ -254,7 +254,7 @@
         }
 
         .m-card_footer {
-            margin-top: ( @card_img_height / 2 ) + @grid_gutter-width;
+            margin-top: ( @card_img_height / 2.2 ) + @grid_gutter-width;
```

Comment:
I'm going to change this to subtract 6px so it ends up a whole pixel value and is not dependent on the image height.